config/config.go

```diff
@@ -472,31 +472,32 @@ func (cfg RPCConfig) IsTLSEnabled() bool {
 
 // P2PConfig defines the configuration options for the Tendermint peer-to-peer networking layer
 type P2PConfig struct {
-	RootDir string `mapstructure:"home"`
```

Comment:
I don't like the new layout. We should probably ignore maligned linter here to preserve logical grouping rather than optimize memory layout